featuremanagement/_featuremanager.py

```diff
@@ -116,32 +114,28 @@ def _is_targeted(context_id):
 
         return (context_marker / (2**32 - 1)) * 100
 
-    def _assign_variant(self, feature_flag, **kwargs):
+    def _assign_variant(self, feature_flag, targeting_context):
```

Comment:
I'd expect this function to set `evaluation_event.variant` and return the `evaluation_event`, rather than a double return. 

Looking at it's usage- I'd actually expect evaluation_event to be passed in as a param and modified directly.

Reply:
So, I updated part of this. The `evaluation_event` has been removed. As for the part before that. This method just find the name of the variant. It doesn't get attached to the `evaluation_event` until after we return and check that we returned a none `None` value. See line 241.